nox/sessions.py

```diff
@@ -361,6 +395,9 @@ def install(self, *args: str, **kwargs: Any) -> None:
         if not args:
             raise ValueError("At least one argument required to install().")
 
+        # Escape args that should be: not needed and would make pip raise an InvalidRequirement
```

Comment:
You can remove this instead of leaving it commented out.

Reply:
Done. This was for maintainability - but instead of writing this here, I edited the comment in the conda part so that future readers are not tempted to apply the same escaping procedure to the pip section.